CellEditorView.pde

```diff
@@ -1,35 +1,92 @@
 class CellEditorView extends ViewBase {
     CellModel cellModel;
 
-    ArrayList<CellEditorViewClient> clients = new ArrayList<CellEditorViewClient>();
+    PFont font;
 
+    final float CODONS_Y_POS = 380f;
+    final float CODONS_SPACING = 5f;
 
     CellEditorView(ViewBase parentView, CellModel cellModel) {
         super(parentView);
 
         this.cellModel = cellModel;
 
-        this.boundsRect = new Rectangle2D.Float(0, 0, 200, 300);
         this.shouldClip = true;
-    }
 
-    void registerClient(CellEditorViewClient client) {
-        if(!clients.contains(client)) {
-            clients.add(client);
-        }  
+        font = createFont("courrier.dfont", 24);
```

Comment:
can you move this to the to setup function

Reply:
Already done